src/gitUtils.ts

```diff
@@ -1,21 +1,43 @@
 import { exec } from "@actions/exec";
 import { execWithOutput } from "./utils";
 
-export const setupUser = async () => {
+export const setupUser = async (githubUserName: string, githubUserEmail: string) => {
   await exec("git", [
     "config",
     "--global",
     "user.name",
-    `"github-actions[bot]"`,
+    githubUserName,
   ]);
   await exec("git", [
     "config",
     "--global",
     "user.email",
-    `"github-actions[bot]@users.noreply.github.com"`,
+    githubUserEmail,
   ]);
 };
 
+export const setupCommitSigning = async (gpgPrivateKey: string) => {
+  await exec("apk", [
```

Comment:
is apk even available by default on ubuntu? why the Crowdin action doesn't have to do this installation at all?

Reply:
The crowdin action does this installation in the [`Dockerfile`](https://github.com/crowdin/github-action/blob/master/Dockerfile)

What would the alternative be? Have a prerequisite that if you want commit signing the image you're using should have it installed already?

Reply:
> What would the alternative be? Have a prerequisite that if you want commit signing the image you're using should have it installed already?

Yes, this sounds reasonable to me - I agree that it would be better if we could handle this automatically, but installing tools seems out of the scope of this simple node.js action. Note that most of our users use the default ubuntu images but it's technically possible that this action gets executed in different contexts too - so I'm afraid that we would always fail to recognize a particular environment and thus we'd fail to install this correctly on the user's behalf.

Reply:
Agreed, I'll remove this line then and add something to the `README`